### ISSUE TITLE
Black screen + fatal error after updating launcher

### ISSUE THREAD

**Issue by seenasa** [seathasky/FF14-MAC_ModSupport#14]
Hi @seathasky i just tried launching the game and got the same blackscreen fatal error after seeing the notification that there had been an update to the launcher. Not sure what to do but im reinstalling using the new m1 specific launcher. Please advise, ty!

Now there is 2 ffxiv launcher bottles and neither seem to work. Pressing the login button is unresponsive at the login screen

_Originally posted by @ssalehian0 in https://github.com/seathasky/FF14-MAC_ModSupport/issues/12#issuecomment-965990386_

**Comment by seenasa** [comment]
<img width="907" alt="Screen Shot 2021-11-10 at 9 22 19 PM" src="https://user-images.githubusercontent.com/15369952/141242144-556bd569-3c6b-43f0-944e-c863366c5048.png">

**Comment by kailred** [comment]
to add to this, i am encountering the issue that states "The Square Enix account cannot play Final Fantasy XIV" as mentioned in the other thread I just commented on as well. however, I did not use the new alternative install you created this evening, but was using the same bottle from earlier that ran successfully for the entire day. i thought that might be the issue, so I created a fresh bottle using your new m1 installer and the issue persists. I'm not entirely sure it's related to anything on your end, but rather something from XIVLauncher or Square Enix themselves. Or maybe I'm just being dramatic!
